ca_driver/src/create_driver.cpp

```diff
@@ -255,6 +263,16 @@ void CreateDriver::publishOdom()
   odom_pub_.publish(odom_msg_);
 }
 
+void CreateDriver::publishJointState() {
+    // Publish joint states
+    joint_state_msg_.header.stamp = ros::Time::now();
+    joint_state_msg_.position[0] = robot_->getLeftWheelDistance() / (create::util::CREATE_2_WHEEL_DIAMETER / 2);
```

Comment:
Technically, I don't think it is any more efficient, since it should be computed at compile time because both operands are constants, but it looks cleaner to use a variable.